Assets/AWSIM/Scripts/Editor/RandomTraffic/Environments/LaneletLoader/LaneletBoundsLoader.cs

```diff
@@ -0,0 +1,192 @@
+// Copyright 2023 Tier4.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using AWSIM.Lanelet;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using GeometryUtility = AWSIM.Lanelet.GeometryUtility;
+
+namespace AWSIM.RandomTraffic
+{
+    /// <summary>
+    /// A class that provides capability to convert Lanelet2 data into environment components.
+    /// </summary>
+    public class LaneletBoundsLoader
+    {
+        [System.Serializable]
+        public struct WaypointSettings
+        {
+            [SerializeField, Range(0.1f, 2f)]
+            [Tooltip("Resolution of resampling. Lower values provide better accuracy at the cost of processing time.")]
+            private float resolution;
+            [SerializeField, Range(5f, 100f)]
+            [Tooltip("Minimum length(m) between adjacent points.")]
+            private float minDeltaLength;
+            [SerializeField, Range(5f, 30f)]
```

Comment:
angle thresh should be 1deg ~30deg
```suggestion
            [SerializeField, Range(1f, 30f)]
```